graphein/protein_ligand/__init__.py

```diff
@@ -0,0 +1,22 @@
+"""Protein graph construction module."""
+from .config import *
+from .edges import *
+from .graphs import *
+from .utils import *
+from .visualisation import (
+    plot_distance_landscape,
+    plot_distance_matrix,
+    plot_protein_structure_graph,
+    plotly_protein_structure_graph,
+)
+
+try:
+    from .visualisation import plot_chord_diagram
+except ImportError:
+    pass
```

Comment:
We should use the import message util from `graphein.utils.utils.import_message`